mars/tensor/expressions/tests/test_reshape.py

```diff
@@ -48,3 +49,18 @@ def testReshape(self):
         self.assertEqual(calc_shape(b), b.shape)
         self.assertEqual(calc_shape(b.chunks[0]), b.chunks[0].shape)
         self.assertEqual(tuple(sum(s) for s in a.nsplits), (10, 30, 20))
+
+    def testShuffleReshape(self):
+        from mars.tensor import reshape
+        a = ones((31, 27), chunk_size=10)
+        b = reshape(a, (27, 31), _reshape_with_shuffle=True)
```

Comment:
`_reshape_with_shuffle` looks pretty weird, if it's only for tests purpose, just modify the op property in the tests looks fine to me.